src/+otp/+vanderpol/+presets/ExtendedPrecision.m

```diff
@@ -0,0 +1,37 @@
+classdef ExtendedPrecision < otp.vanderpol.VanderpolProblem
+    methods
+        function obj = ExtendedPrecision(epsilon)
+            if nargin < 1
+                epsilon = 1e-6;
+            end
+            
+            cls = class(epsilon);
+            
+            coeffs = [ ...
+                cast(199944165973, cls) / cast(3486784401, cls) * cast(7499951716, cls) / cast(4782969, cls), ...
```

Comment:
I had to express `1532057748884056/16677181699666569` in terms of numbers that fit into a double.  Perhaps there is a more concise way to do this.